scanner/src/test/assets/scanoss/filesToScan/Semver4jRange.java

```diff
@@ -0,0 +1,92 @@
+package com.vdurmont.semver4j;
```

Comment:
It makes me feel a bit uneasy with committing third-party code here. Can't we simply copy (some old state of) some ORT code itself?

Reply:
Unfortunately, you are right :(
I will change the examples and adapt the code, it will take some time because I need to recapture the requests with wiremock.